### PR TITLE
Use Highmaps for map rendering instead of GoogleCharts

### PR DESCRIPTION
**Pull request by joahim**
This PR migrates the map visualization from GoogleCharts to Highmaps. It also unifies/cleans the onLoad event creation.

### CODE REVIEW

**Comment by joahim** [comment]
@stefanb I moved the maps to `public/maps/` so that they have a stable URL (no hash added by the Webpack).

File: src/visualizations/App.fsproj [hunk 1]
```diff
@@ -5,7 +5,6 @@
     <ItemGroup>
         <Compile Include="Types.fs" />
         <Compile Include="Highcharts.fs" />
-        <Compile Include="GoogleCharts.fs" />
         <Compile Include="Utils.fs" />
```
Comment (stefanb) [inline src/visualizations/App.fsproj:8]: For those using visual studio add it to the project file:
```suggestion
        <Compile Include="GoogleCharts.fs" />
        <Compile Include="Utils.fs" />
```

File: src/visualizations/AgeGroupsChart.fs [hunk 2]
```diff
@@ -315,93 +315,94 @@ let renderChartOptions
...
-              dataLabels = pojo
-                {| enabled = true
-                   formatter = fun () -> valuesLabelFormatter jsThis?y
-                   align = "left"
-                   style = pojo {| textOutline = false |}
-                   padding = 10 |}
-              data = chartData.FemaleValues
-               |}
-       |]
+    {| Highcharts.optionsWithOnLoadEvent "covid19-age-groups" with
```
Comment (stefanb) [inline src/visualizations/AgeGroupsChart.fs:318]: Does this line belong into this PR? (everything below was just re-indented)

**Comment by stefanb** [comment]
Fixes #176

File: src/visualizations/AgeGroupsChart.fs (hunk 2 above)
Comment (joahim) [reply src/visualizations/AgeGroupsChart.fs:318]: It does not belong directly but it does :-) to use the onLoad event trigger in the Map consistently across all visualizations. Indentation is important (anonymous record vs anonymous record extension).

**Pull request merged**